x-pack/plugins/apm/server/routes/storage_explorer/has_storage_explorer_privileges.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { every } from 'lodash';
+import { ApmPluginRequestHandlerContext } from '../typings';
+import { Setup } from '../../lib/helpers/setup_request';
+
+export async function hasStorageExplorerPrivileges({
+  context,
+  setup,
+}: {
+  context: ApmPluginRequestHandlerContext;
+  setup: Setup;
+}) {
+  const {
+    indices: { transaction, span, metric, error },
+  } = setup;
+
+  const apmIndices: string[] = [];
+  [transaction, span, metric, error].forEach((indicesForProcessorEvent) => {
```

Comment:
`hasPrivileges` API call needs an array of indices, is there any helper that does this?